@nav-frontend/link-styles/src/index.css

```diff
@@ -0,0 +1,40 @@
+.navds-link {
+  color: var(--navds-color-text-link);
+  background: none;
+  text-decoration: underline;
+  cursor: pointer;
```

Comment:
```
text-decoration: underline;
cursor: pointer;
```
er vel default for `a`? Vet du hvorfor vi setter det her?

Reply:
Vil anta det er for å sikre en standard, da nettlesere gjerne har egne implementasjoner på slikt. Er ikke tilfellet for `<a>` i chrome/firefox/safari så er kanskje overforsiktig.

Reply:
Jeg tenker vi kan lene oss på normalize.css her. Ser ikke ut som de gjør så mye med lenker heller (https://necolas.github.io/normalize.css/8.0.1/normalize.css):
```
/**
 * Remove the gray background on active links in IE 10.
 */

a {
  background-color: transparent;
}
```

Reply:
Ser de fleste designsystem setter `text-decoration: underline;` selv så tenker vi beholder den, men fjerner background og pointer.